main.nf

```diff
@@ -20,7 +20,7 @@
 */
 
 // Using DSL-2
-nextflow.enable.dsl=2
+nextflow.preview.dsl=2
```

Comment:
Would you mind pushing another commit to change this back to `nextflow.enable.dsl = 2`?

Reply:
New push with this change